wiki/People/Tournament_Committee/en.md

```diff
@@ -26,7 +33,111 @@ Their responsibilities include:
+
+- Determine if the issues brought up in the report break any of the Official Support criteria
+- If they do break those criteria, determine whether the severity of those infringements could potentially warrant withholding a badge from the tournament or giving a tournament staffing ban to any of the staff of that tournament
+- If they are severe enough to warrant such action, independently investigate the events detailed in the report to ensure the report’s veracity and gather additional context
+- Reach out to the infringing party, e.g. the tournament organiser, to get further information on the events detailed in the report
+- If after this investigation is over and the Tournament Committee is still considering handing out relevant punishment, hold an anonymous vote to determine whether to do so and the severity of such punishment
+
+### News Posts
+
+When an email requesting registration and conclusion news posts for a tournament is received, a thread is automatically created with an embed message of the email’s text. Any Tournament Committee member who sees this will ping the rest of the group for visibility. The committee will then check if the tournament is eligible based on the requirements on the Official Support wiki page before moving forward. If the tournament is eligible and the consensus in discussion around the tournament is positive after 48 hours, the Tournament Committee will forward the request to the news post team and approve the request. If the tournament does not pass one of these steps, the request will be denied.
+
+### Top Three Badge Support
+
+If a tournament host believes their tournament has achieved the highest level of production quality and organisation possible, they can request to receive badge prizes for all three finalists in their tournament. If a host requests to be awarded with this distinction, then in addition to the typical steps for tournament review as listed above, the assigned members will also notify the rest of the group of the request for top three badge support. Based on discussion around the tournament and its quality, either the request will be denied, or an anonymous vote will be called for the tournament to receive top three badge support.
+
+### Public Communication
+
+Public communication by the Tournament Committee is handled through several channels. The purposes for these sometimes overlap, but all are official sources of information kept up-to-date by the Tournament Committee.
+
+The Tournament Committee maintains an [account on X](https://twitter.com/osu_tcomm) (formerly Twitter) where the committee broadcasts major changes and updates of various types. Not all changes are communicated here, but many of the most important changes are communicated and clarified through this channel.
```

Comment:
```suggestion
The Tournament Committee maintains an [account on X](https://x.com/osu_tcomm) where the Committee broadcasts major changes and updates of various types. Not all changes are communicated here, but many of the most important changes are communicated and clarified through this channel.
```

Reply:
Already changed twitter to x, but committee by itself should not be capitalized.

Reply:
Also just calling it X is enough to confuse most people, many large corporations use X (formerly Twitter) for clarity following the rebrand.

Reply:
Sure, I'm indifferent on that.